### PR TITLE
Creating branch for overview of Astra DB updates

### PR DESCRIPTION
**Pull request by SonicDMG**
This branch contains updates per feedback received in https://datastax.jira.com/browse/APRT-92

### CODE REVIEW

File: overviewOfAstraDB/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Overview
-Understand Astra DB and how it differs from its underlying technology, Apache Cassandra®.
+Astra DB is a serverless database service (DBaaS) built on Apache Cassandra®.
```
Comment (msmygit) [inline overviewOfAstraDB/README.md:2]: Why not specify it in full like `..database-as-a-service...(DBaaS)...`?

Comment (msmygit) [reply overviewOfAstraDB/README.md:2]: Also please remove the extra space after the period that got introduced here.

Comment (SonicDMG) [reply overviewOfAstraDB/README.md:2]: We could potentially do that @msmygit, seems fine to me. @simonfraserduncan, do you have any thoughts on this?

Comment (SonicDMG) [reply overviewOfAstraDB/README.md:2]: So, something like:
"Astra DB is a serverless database-as-a-service (DBaaS) built on Apache Cassandra®."

Comment (simonfraserduncan) [reply overviewOfAstraDB/README.md:2]: I think that's fine. Many evaluating Astra will be familiar with DBaaS as an acronym, and for those who aren't, we're providing the explanation (which is more than what we offer on https://www.datastax.com/products/datastax-astra right now).

Comment (SonicDMG) [reply overviewOfAstraDB/README.md]: Thank you

**Review by simonfraserduncan — APPROVED** [review]
Agree with this formatting 👏

**Pull request merged**